src/state/subscriptions/sagas.js

```diff
@@ -150,6 +152,7 @@ function* subscribeFlow(action) {
         );
       } catch (e) {
         console.error('Error subscribing to ', action.resource, e);
+        yield put(actionCreators.subscriptionLoaded(action.resource));
```

Comment:
Can we add a fat "TODO" in here about handling error state differently than "loaded"?